src/RedisClusterOptions.php

```diff
@@ -248,6 +250,19 @@ public function getLibOption(int $option, mixed $default = null): mixed
         return $this->libOptions[$option] ?? $default;
     }
 
+    public function getUsername(): ?string
+    {
+        return $this->username;
+    }
+
+    /**
+     * @psalm-param non-empty-string $username
```

Comment:
No need to use `psalm-` prefix anymore. 
Nowadays, IDEs have support for this kind of narrowed types and thus it won't be necessary anymore.